### PR TITLE
Fixed a typo in Resources.resx

### PR DESCRIPTION
**Pull request by melytc**

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/Resources.resx [hunk 1]
```diff
@@ -181,9 +181,9 @@ If this file or folder does not appear in Solution Explorer, then it is not curr
   <data name="ProjectLoadedWithWrongProjectType" xml:space="preserve">
     <value>The project file '{0}' cannot be opened. 
 
-This project was loaded using the wrong project type, likely as a result of renaming the project extension outside of Visual Studio. Remove and readd the project to have the solution pick up the right project type.</value>
+This project was loaded using the wrong project type, likely as a result of renaming the project extension outside of Visual Studio. Remove and read the project to have the solution pick up the right project type.</value>
```
Comment (davkean) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/Resources.resx:184]: This should be "re-add", not "read".

Comment (davidwengier) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/Resources.resx:184]: This is talking about removing and then adding back the project, not reading the project :P

```suggestion
This project was loaded using the wrong project type, likely as a result of renaming the project extension outside of Visual Studio. Remove and re-add the project to have the solution pick up the right project type.</value>
```

Comment (melytc) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/Resources.resx:184]: That makes sense, thanks for clarifying it.

**Comment by melytc** [comment]
It got localized as `read`, at least in the Spanish and French ones. #6106

**Comment by davkean** [comment]
You'll need to regenerate the strings by building at the command-line.

**Pull request merged**